CHANGELOG.md

```diff
@@ -59,6 +59,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed: When Progressive Grapple is enabled, it will now show `2 shuffled copies` rather than `Shuffled` for better consistency.
 - Changed: A proper error message is displayed when mono is not found, when exporting a game on macOS and Linux.
 
+### Metroid Prime 3: Corruption
+
+#### Logic Database
+
+- Added: replaced the video linked for Command Courtyard with a more accurate video.
```

Comment:
"Changed" instead of "Added" seems more appropriate for this

Reply:
We don't need changelogs for Prime 3

Reply:
Felt weird adding one, but I just did what I was taught haha. Deleted.